osu.Game.Tests/Visual/Editing/TestSceneSubmissionStageProgress.cs

```diff
@@ -43,5 +52,92 @@ public void TestAppearance()
+
+            AddStep("prepare", () =>
+            {
+                Child = new Container
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Size = new Vector2(1),
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Child = new FillFlowContainer
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        Size = new Vector2(0.8f),
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                        Direction = FillDirection.Vertical,
+                        Spacing = new Vector2(0, 5),
+                        Children = new Drawable[]
+                        {
+                            stages[0] = new SubmissionStageProgress
+                            {
+                                Anchor = Anchor.Centre,
+                                Origin = Anchor.Centre,
+                                StageDescription = "Export...",
+                                StageIndex = 0
+                            },
+                            stages[1] = new SubmissionStageProgress
+                            {
+                                Anchor = Anchor.Centre,
+                                Origin = Anchor.Centre,
+                                StageDescription = "CreateSet...",
+                                StageIndex = 1
+                            },
+                            stages[2] = new SubmissionStageProgress
+                            {
+                                Anchor = Anchor.Centre,
+                                Origin = Anchor.Centre,
+                                StageDescription = "Upload...",
+                                StageIndex = 2
+                            },
+                            stages[3] = new SubmissionStageProgress
+                            {
+                                Anchor = Anchor.Centre,
+                                Origin = Anchor.Centre,
+                                StageDescription = "Update...",
+                                StageIndex = 3
+                            },
+                            cardContainer = new Container
+                            {
+                                Anchor = Anchor.Centre,
+                                Origin = Anchor.Centre,
+                            }
+                        }
+                    }
+                };
+
+                completeSample = audio.Samples.Get(@"UI/bss-complete");
+            });
+
+            for (int i = 0; i < stages.Length; i++)
+            {
+                int step = i;
+                AddStep($"{step}: not started", () => stages[step].SetNotStarted());
+                AddStep($"{step}: indeterminate progress", () => stages[step].SetInProgress());
+                AddStep($"{step}: 70% progress", () => stages[step].SetInProgress(0.25f));
```

Comment:
```suggestion
                AddStep($"{step}: 25% progress", () => stages[step].SetInProgress(0.25f));
```

or maybe even keep both

```suggestion
                AddStep($"{step}: 25% progress", () => stages[step].SetInProgress(0.25f));
                AddStep($"{step}: 70% progress", () => stages[step].SetInProgress(0.7f));
```